eventmesh-common/src/main/java/org/apache/eventmesh/common/config/ConfigMonitorService.java

```diff
@@ -34,8 +34,7 @@ public class ConfigMonitorService {
 
     private final List<ConfigInfo> configInfoList = new ArrayList<>();
 
-    private final ScheduledExecutorService configLoader = ThreadPoolFactory
-        .createSingleScheduledExecutor("eventMesh-configLoader-");
+    private final ScheduledExecutorService configLoader = ThreadPoolFactory.createSingleScheduledExecutor("eventMesh-configLoader");
```

Comment:
This is a very nice modification point to help keep the consistency of EM thread naming; you can modify the name and description of the PR; at the same time, EM about the thread naming prefix, there is also this naming style, the code is here: org/apache/eventmesh/runtime/core/protocol/http/push/ HTTPMessageHandler.java:52